src/data/markdown/translated-guides/en/01 Get started/02 Installation/01 Troubleshooting.md

```diff
@@ -19,20 +19,15 @@ This example is for Debian/Ubuntu and derivatives. Consult your distribution's d
 
 Some Ubuntu and Debian users might run into an `apt-key` deprecation warning while adding k6's repository key to their system's keyring:
 
-> `Warning: apt-key is deprecated. Manage keyring files in trusted.gpg.d instead (see apt-key(8))`
+```bash
+Warning: apt-key is deprecated. Manage keyring files in trusted.gpg.d instead (see apt-key(8))`
+```
 
-To avoid this and be future-proof, users should delete the existing `security@k6.io` repository key on their system, and update their sources list accordingly.
+To avoid this and be future-proof, users should delete the existing `security@k6.io` repository key on their system with the following command:
 
 ```bash
 # delete existing key
 sudo apt-key del k6
-
```

Comment:
:thinking: I think we might want to remove this `apt-key` section altogether. This was an error users experienced when the instructions mentioned using `apt-key`, but since we no longer mention it, there's no reason to list this here.

Reply:
@imiric, 

Given this reasoning, we might want to remove the `Error importing k6's GPG key` section if `sudo gpg -k` to the recommended instructions.